### ISSUE TITLE
Trained loras barely work on ComfyUI

### ISSUE THREAD

**Issue by ClaraSanders** [filliptm/ComfyUI-FL-AceStep-Training#14]
As many people pointed out before me, trained Loras are super weak in style, regardless of the steps, learning rate, or number of songs in the dataset. You can overtrain a Lora with 10k epochs, and it will be like putting a normal Lora at 0.3. Do we need some kind of special settings or a different workflow than the one used by RyanOntheInside?
